bin/azure-console

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+ENVIRONMENT_NAME=$1
+
+case $ENVIRONMENT_NAME in
+  "development")
+    SUBSCRIPTION_ID="8655985a-2f87-44d7-a541-0be9a8c2779d"
+    RESOURCE_GROUP_PREFIX="s118d01"
+    ;;
+  "production")
+    SUBSCRIPTION_ID="88bd392f-df19-458b-a100-22b4429060ed"
+    RESOURCE_GROUP_PREFIX="s118p01"
+    ;;
+  *)
+    echo "Could not find a known environment with the name: $ENVIRONMENT_NAME"
+    exit 1
+    ;;
+esac
+
+if [[ $ENVIRONMENT_NAME == "production" ]]; then
+  echo "**********************************************************************************************************************"
+  echo "You will need to make a PIM request to access the console"
+  echo "Visit https://portal.azure.com/#blade/Microsoft_Azure_PIMCommon/ActivationMenuBlade/azurerbac"
+  echo "and activate the 'Contributor' role for the 's118-teacherpaymentsservice-production' environment."
+  echo "Once this has been approved, press enter to continue"
+  echo "**********************************************************************************************************************"
+
+  read -p "" -r
+
+  echo "Logging out of Azure and back in to grab new tokens"
+  if az account show > /dev/null; then
```

Comment:
This doesn't seem to behave correctly when not logged in to the Azure CLI - I tried it locally and got `The subscription of '8655985a-2f87-44d7-a541-0be9a8c2779d' doesn't exist in cloud 'AzureCloud'.` It worked fine after a manual `az login`.

Reply:
I've added an explicit `az login` to the script that should fix this